tests/test_dynamical_decoupling.py

```diff
@@ -437,6 +437,193 @@ def test_conversion_of_z_pi_2_pulses_at_extremities():
         [2.5e-2, _duration - 2* 2.5e-2, 2.5e-2]))
 
 
+
+def test_conversion_of_pulses_with_arbitrary_rabi_rotations():
+    """
+    Tests if the method to convert a DDS to driven controls handles properly
+    Y pulses with rabi rotations that assume arbitrary values between 0 and pi.
+    """
+    _number_of_pulses = 30
+    _duration = _number_of_pulses * 1.
+    _offsets = np.linspace(0.5, _duration-0.5, _number_of_pulses)
```

Comment:
When comparing to the expected durations defined below, it took a little while to convince myself about what the spacing is here. I think I would have found it simpler to understand if this was just `0.5 + np.arange(_number_of_pulses)`.